packages/zowe-explorer-ftp-extension/package.json

```diff
@@ -52,9 +52,9 @@
     "vscode": "^1.79.0"
   },
   "dependencies": {
-    "@zowe/zos-files-for-zowe-sdk": "8.0.0-next.202408291544",
+    "@zowe/zos-files-for-zowe-sdk": "8.0.0-next.202408301809",
     "@zowe/zos-ftp-for-zowe-cli": "3.0.0-next.202403191358",
```

Comment:
3.0.0-next.202403191358 is pretty old, is there a reason against bumping @zowe/zos-ftp-for-zowe-cli to 3.0.0-next.202407311518?

Reply:
No objections here 😋 
it will probabyl be a good idea since there are a few fixes ready to go on the FTP vNext version 😋